packages/grid/_modules_/grid/hooks/virtualization/useVirtualRows.ts

```diff
@@ -206,6 +206,8 @@ export const useVirtualRows = (
   const scrollingTimeout = React.useRef<any>(0);
   const onScroll: any = React.useCallback(
     (event: any) => {
+      if (event.target.scrollLeft < 0) return;
```

Comment:
P.S. now I get why ag-Grid RTL was broken last month. Chrome 85 released an update the made them adapt to the latest spec for handling scrollLeft on rtl. On Material-UI side, we didn't see any impact because we feature detect the mode. ag-Grid was using a user-agent logic. I guess it's a reminder that testing user-agent isn't as great as doing feature support detection.